src/optimize.jl

```diff
@@ -18,9 +18,9 @@ function lookup_stmt(stmts::Vector{Any}, @nospecialize arg)
         if isa(q, QuoteNode) && isa(q.value, Symbol)
             mod = lookup_stmt(stmts, arg.args[2])
             if isa(mod, GlobalRef)
-                mod = getproperty(mod.mod, mod.name)
+                mod = @invokelatest getproperty(mod.mod, mod.name)
             end
-            isa(mod, Module) && return getproperty(mod, q.value)
+            isa(mod, Module) && return @invokelatest getproperty(mod, q.value)
```

Comment:
```suggestion
            isa(mod, Module) && return @invokelatest getglobal(mod, q.value)
```
I don't think this change will significantly improve performance, but since we've already raised the minimum compatibility to v1.10, there's no longer a need to call `getproperty`.